### DISCUSSION TITLE
建议讨论进行下分组

### DISCUSSION THREAD

**Discussion by iGao101** [Tencent/libpag#1982] [Ideas] [answered]
建议讨论进行下分组，Q&A现在有好几十页的内容，需要能做一个分组，比如android、ios、flutter等，这样子根据分组检索会方便一些

**Comment by hubeishuaige** [comment]
哎  强烈建议PAG团队整个讨论组，遇到问题，不能及时沟通，愁死了。

**Reply by kevingpqi123** [reply]
目前的大环境都是降本增效，对接群已停止维护，为了提高对接效率以及沉淀问题，所有的问题提问一律通过 github discussion，遇到问题的时候建议先检索下是否类似问题，你想要的实时对接社区版不提供
如果自身业务对于实时对接要求比较强烈，可以接入 PAG 企业版

**Comment by kevingpqi123** [comment, marked as answer]
后续可以添加，使用层面更多的是关键词检索，对于 pag 而言，主体实现都在 C++ 层，各平台的 API 和能力保持一致性，使用层面和平台没有太大关系

**Reply by iGao101** [reply]
好的，感谢
